components/Navigation/index.js

```diff
@@ -0,0 +1,21 @@
+import Link from "next/link";
+import styled from "styled-components";
+import "material-symbols";
+
+const StyledList = styled.ul`
+  display: flex;
+  justify-content: space-around;
+  width: 100%;
+  height: 65px;
+  position: fixed;
+  bottom: 0;
+  font-size: 10px;
+  padding: 3px;
+  align-items: center;
+  background-color: lightgray;
+  z-index: 21;
+`;
+
+export default function Navigation() {
+  return <StyledList></StyledList>;
```

Comment:
It looks like the navigation is empty!? If so, it should not be included in a PR. Always try to add only finished feature/code into a PR.

Reply:
Ok, yes i was a little confused and i already had it before, next time i will only add finished code in the PR.